### PR TITLE
Adding doctor role + updating the tests

### PR DESCRIPTION
**Pull request by psykzz**
Added the doctor role to the game.
Along with updating the tests to pass.

### CODE REVIEW

File: test/game.js [hunk 1]
```diff
@@ -135,64 +151,99 @@ describe("new Game()", function() {
...
+      doctor.socket.ons[game.getCurrentIdentifier()]("doesnotexist");
+
+      // Should have been acknowledged, and asked again
+      assert.equal(doctor.socket.emits[0].type, "rfa");
+      assert.equal(doctor.socket.emits[1].type, "ack");
+      assert.equal(doctor.socket.emits[2].type, "rfa");
+      done();
+    });
+
+    it("should register doctors vote on existing player", function(done) {
```
Comment (Neamar) [inline test/game.js:181]: Doctor?

**Review by Neamar — COMMENTED** [review]
Shouldn't the doctor be able to heal only once per game?

File: test/game.js (hunk 1 above)
Comment (psykzz) [reply test/game.js:181]: Are you complaining about plural here or something else? This is correct to me

**Comment by psykzz** [comment]
Doctor typically can heal around 3 times a game, and only once on themselves.
Limits however are for later on.

**Comment by Neamar** [comment]
3 times is way to much. But yeah let's discuss this later

**Pull request merged**